easybuild/easyblocks/i/impi.py

```diff
@@ -295,7 +295,7 @@ def make_module_req_guess(self):
                 manpath = os.path.join(mpi_subdir, 'man')
 
                 if self.cfg['ofi_internal']:
-                    libfabric_dir = os.path.join('mpi', '2021.1.1', 'libfabric')
+                    libfabric_dir = os.path.join('mpi', 'latest', 'libfabric')
```

Comment:
Ouch, stupid bug...

We should use `self.version` here though, I prefer not relying on symlinks if we can avoid it...

Reply:
see https://github.com/ComputeCanada/easybuild-easyblocks/pull/7